UPGRADE.md

```diff
@@ -0,0 +1,32 @@
+# Upgrade to 2.0
+
+## Breaking Change: DBAL 3
+
+The most significant change in version 2.0 is using doctrine/dbal 3. You should review their upgrade guide in addition to this one: [https://github.com/doctrine/dbal/blob/3.5.x/UPGRADE.md](https://github.com/doctrine/dbal/blob/3.5.x/UPGRADE.md).
```

Comment:
Tag/branch is removed. Maybe try commit https://github.com/doctrine/dbal/blob/bd54f5043eaff656b314037bf285d8b7f1c311b8/UPGRADE.md (3.6 commit)